remote/gogs/helper.go

```diff
@@ -66,13 +66,29 @@ func buildFromPush(hook *PushHook) *model.Build {
 		hook.Repo.Url,
 		fixMalformedAvatar(hook.Sender.Avatar),
 	)
+
+	eventType := "Undefined"
+	message := "No message"
+
+	switch {
+	case hook.RefType == "tag":
+		eventType = model.EventTag
+		message = "Create Tag " + hook.Ref
+	case hook.RefType == "branch":
+		eventType = model.EventPush
+		message = "Create Branch " + hook.Ref
+	default:
```

Comment:
With more test, I see that gogs works like github, two builds are triggered. Yuo are true, branch creation event should definitiely be ignored